### ISSUE TITLE
Imagenet classification with deep convolutional neural networks

### ISSUE THREAD

**Issue by Soyeon-Cha** [GDSC-Ewha-4th/Study-paperReview#54]
날짜 : 2022.11.14.~2022.11.21.
논문명 : Krizhevsky, Alex, Ilya Sutskever, and Geoffrey E. Hinton. "Imagenet classification with deep convolutional neural networks." Advances in neural information processing systems. 2012. [pdf] (AlexNet, Deep Learning Breakthrough)

- 많은 수의 고해상도 이미지를 클래스로 분류하기 위해 대규모의 심층 convolutional neural network를 이용
- dropout 방식 사용

#### Dataset
- ImageNet은 약 22,000개의 카테고리에 속하는 1,500만 개 이상의 레이블이 지정된 고해상도 이미지 dataset
- ImageNet은 보통 top-1과 top-5 오류율을 보고함
- ImageNet과는 달리 여기서는 고정된 해상도의 입력값을 이용하고 가운데의 RGB 값으로 네트워크 학습시켰음

#### Architecture
- ReLU Nolinearity
  - Rectified Linear Units(ReLUs) : 비선형성을 가진 뉴런
  - 기존의 saturating neuron model을 사용했다면 큰 신경망으로 실험할 수 없었을 것

- Training on Multiple GPUs
  - 두 개의 GPU에 네트워크 분산시키고 병렬화
  - 결과 아키텍처는 열이 독립적이지 않다는 점 외에는 columnar CNN의 아키텍처와 비슷함

- Local Response Normalization
  - ReLUs는 saturating을 막기 위한 input normalization이 필요 없음
  - 특정 계층들에 ReLU의 비선형성을 적용

- Ovelapping Pooling
  - overlapping pooling이 있는 모델이 overfit하기 조금 더 어려움

- Overall Architecture
  - 여기의 네트워크는 multinomial 로지스틱 회귀 목표를 최대화함

#### Reducing Overfitting
  - 상당한 overfitting 없이 많은 매개변수를 학습하기는 어려움
  - data Augmentation과 dropout이 이미지 데이터의 overfitting을 줄이는 대표적인 방법

- Data Augmentation
  - label-preserving transformation을 이용하여 인위적으로 dataset을 확대
  

- Dropout
  - 확률 0.5의 숨겨진 뉴런의 출력을 0으로 설정
  - 아키텍처들은 가중치를 공유

#### Results
  - 상위 1위와 상위 5위의 테스트 세트 오류율을 달성
  - CNN은 18.2%의 상위 5개 오류율을 달성
  - 풀링 레이어에 6차 컨볼루션 레이어가 있는 CNN을 학습시켜 16.6%의 오류율

#### Discussion
- 단일 convolution layer가 제거되면 네트워크의 성능이 저하됨
